internal/config/yaml_parser.go

```diff
@@ -34,8 +35,12 @@ const (
 	ERROR   LogSeverity = "ERROR"
 	OFF     LogSeverity = "OFF"
 
-	parseConfigFileErrMsgFormat           = "error parsing config file: %v"
-	MetadataCacheTtlSecsInvalidValueError = "the value of ttl-secs for metadata-cache can't be less than -1"
+	parseConfigFileErrMsgFormat = "error parsing config file: %v"
+)
+
+var (
+	ErrMetadataCacheTtlSecsInvalidValue = errors.New("the value of ttl-secs for metadata-cache can't be less than -1")
```

Comment:
apologies, the previous way is better when compared to defining variables.

Reply:
I thought of creating these Err* variables as const at first, but there is no straightforward way to do it other than defining a utility for it.

Reply:
For now, I'll switch these back to strings and deal with this question later if needed.